src/buildercore/trop.py

```diff
@@ -94,44 +96,58 @@ def rds_security(context):
 #
 
 
-def instance_tags(context):
-    return [
-        ec2.Tag('Name', context['stackname']),
+def instance_tags(context, node=None):
+    tags = [
         ec2.Tag('Owner', context['author']),
+        # hierarchically ordered
         ec2.Tag('Project', context['project_name']),
+        ec2.Tag('Cluster', context['stackname']),
     ]
-
-def ec2instance(context):
+    if node:
+        tags.append(ec2.Tag('Name', '%s--%d' % (context['stackname'], node)))
+        tags.append(ec2.Tag('Node', '%s' % node))
```

Comment:
You're creating a tag called `Node` .. does `Node` have some other special AWS meaning?